aspnetcore/release-notes/aspnetcore-9/includes/trust_dev_cert_linux.md

```diff
@@ -0,0 +1,24 @@
+#### Trust the ASP.NET Core HTTPS development certificate on Linux
+
+On Ubuntu and Fedora based Linux distros, `dotnet dev-certs https --trust` now configures ASP.NET Core HTTPS development certificate as a trusted certificate for:
+
+* Chromium browsers, for example, Google Chrome, Microsoft Edge, Chromium, and Opera.
```

Comment:
I don't believe Opera is Chromium-based.

Reply:
Apparently, they rebased it on Chromium, but I'd still rather not list it.

Reply:
> Apparently, they rebased it on Chromium, but I'd still rather not list it.

I deleted Chromium

Reply:
Sorry, my comment was unclear - I was following up about Opera, which apparently _is_ Chromium-based now.  The text was fine after removing Opera from the list:

> * Chromium browsers, for example, Google Chrome, Microsoft Edge, and Chromium.